src/mlpack/CMakeLists.txt

```diff
@@ -58,6 +57,15 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
   endif()
 endif()
 
+set_target_properties(mlpack PROPERTIES COTIRE_CXX_PREFIX_HEADER_INIT "prereqs.hpp")
+cotire(mlpack)
+
+get_target_property(_mlpackPrefixHeader mlpack COTIRE_CXX_PREFIX_HEADER)
```

Comment:
Is this needed?  I don't think `_mlpackPrefixHeader` is used anywhere (maybe I missed it?).

Reply:
No it's not, tried using it for the bindings / cli executables.